.github/ISSUE_TEMPLATE/refactor_request.yaml

```
@@ -0,0 +1,15 @@
+name: Refactor request
+description: Suggest a refactoring for this project
+title: "[Refactor]: <title>"
+labels: ["refactor"]
+projects: ["SE-UUlm/2"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: What do you think needs to change? Please describe
+      description: A clear and concise description of what the problem is.
+      placeholder: The service X should return Y instead [...]
```

```suggestion
      placeholder: The method [...| should return [...] instead of [...]
```

Actually we only has one "service" and i think _call_ or _method_ are more correct?